File: sariska-media-swift/sariska-media-swift/Participant.swift

import Foundation

public class User {
  
  public let name: String
  
  public let id: String

  public let avatar: String
  
  public let email: String

    init(name: String = "", id: String = "", avatar: String = "", email: String = "") {
    self.name = name
    self.id = id
    self.avatar = avatar
    self.email = email
  }
}

File: android/app/src/main/java/org/sariska/sdk/Participant.java
package org.sariska.sdk;

import com.facebook.react.bridge.ReadableMap;

class Participant {

    public String id;

    public String displayName;

    public String avatar;

    public String email;

    public String name;

    public boolean isLocal;

    public String role;

    public Participant(ReadableMap readableMap) {
        this.id = readableMap.getString("id");
        this.displayName = readableMap.getString("displayName");
        this.avatar = readableMap.getString("avatar");
        this.email = readableMap.getString("email");
        this.name = readableMap.getString("name");
        this.isLocal = readableMap.getBoolean("isLocal");
        this.role = readableMap.getString("role");
    }
}
